### PR TITLE
Update week3Documentation.md

### PR DESCRIPTION
**Pull request by RadNi**
new version of typedValidator explained. Typos in visualization.

### CODE REVIEW

File: week3Documentation.md [hunk 1]
```diff
@@ -276,6 +276,25 @@ mkValidator p () () ctx = traceIfFalse "beneficiary's signature missing" signedB
     deadlineReached :: Bool 
     deadlineReached = contains (from $ deadline p) $ txInfoValidRange info
 ```
+**Validator Haskell Template**
+
+In this scenario, since the `mkValidator` now accepts paramiters as its input we have to change validator template. Additionally, at compile time we don't know what is the value of `p`, so it is not possible to compile them alltoghether. One way around is to use the `PlutusTx.applyCode` function and compile both the `p` and `mkValidator` objects separatley, and at the end, combine them alltogether.
```
Comment (adacapo21) [inline week3Documentation.md]: **In this scenario, since the `mkValidator` now accepts parameters**

Comment (adacapo21) [reply week3Documentation.md]: all together

**Review by adacapo21 — COMMENTED** [review]
Resolve typos :)

**Comment by RadNi** [comment]
Resolved. Also, there were other typos that I fixed them :)

**Review by adacapo21 — APPROVED** [review]

**Pull request merged**